### PR TITLE
Fetch/Display Mobile App QR Code

### PR DESCRIPTION
**Pull request by joeyorlando**
**What this PR does**:
Adds support for fetching/displaying QR code that will (eventually) be used to link the mobile app to your OnCall instance.

https://www.loom.com/share/aa12bfd7f5b846459f5976161d4550ad

**Checklist**
- [x] Tests updated
- [x] `CHANGELOG.md` updated
- [ ] Documentation added

### CODE REVIEW

File: .github/workflows/integration_tests.yml [hunk 1]
```diff
@@ -88,3 +88,72 @@ jobs:
               echo 'Expected "1, 1" (alert groups, alerts). They were not created in 30 seconds during this integration test. Something is broken' >&2
               exit 1
           fi
+
+  ui-integration-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```
Comment (joeyorlando) [inline .github/workflows/integration_tests.yml:96]: a lot of this is copy/paste from the previous job.

Ideally we'd be able to create a CI workflow as such:
- deploy-helm-release
  - backend-interations-tests (dependent on parent job)
  - ui-interations-tests (dependent on parent job)

look into [sharing docker containers between jobs](https://github.com/docker/build-push-action/issues/225)

File: grafana-plugin/src/containers/MobileAppVerification/parts/DisconnectButton/index.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+import { Button } from '@grafana/ui';
+
+import WithConfirm from 'components/WithConfirm/WithConfirm';
+
+type Props = {
+  onClick: () => void;
+};
+
+// TODO: right now this shows a confirmation pop-up modal on top of the user settings modal, do we want to maybe change this?
```
Comment (joeyorlando) [inline grafana-plugin/src/containers/MobileAppVerification/parts/DisconnectButton/index.tsx:11]: @teodosii UX related question to point out

File: grafana-plugin/src/containers/UserSettings/parts/index.tsx [hunk 3]
```diff
@@ -139,9 +133,8 @@ export const TabsContent = observer((props: TabsContentProps) => {
         ) : (
           <PhoneVerification userPk={id} />
         ))}
-      {activeTab === UserSettingsTab.MobileAppVerification && (
-        <MobileAppVerification userPk={id} phone={storeUser.unverified_phone_number || '+'} />
-      )}
+      {/* TODO: we should probably hide this tab when a user (ie. Admin) is viewing the user settings for another user. Would it make sense for an Admin to be able to link their mobile app to another user's profile */}
```
Comment (joeyorlando) [inline grafana-plugin/src/containers/UserSettings/parts/index.tsx:136]: general question/comment. should we hide this tab when a user is viewing the settings for another user? I don't think it makes sense to allow user's to link their own mobile app to the profile of another user. Thoughts?

File: grafana-plugin/src/models/user/user.test.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+describe('UserStore.getMobileAppQRCode', () => {
+  test.todo('it makes the proper API call and returns the response');
+});
+
+describe('UserStore.disconnectMobileApp', () => {
+  test.todo('it makes the proper API call and returns the response');
```
Comment (joeyorlando) [inline grafana-plugin/src/models/user/user.test.ts:6]: TODO

**Review by vadimkerr — APPROVED** [review]

File: grafana-plugin/src/models/user/user.ts [hunk 5]
```diff
@@ -82,11 +82,6 @@ export class UserStore extends BaseStore {
     };
   }
 
-  @action
-  getCurrentUser() {
-    return this.items[this.currentUserPk as User['pk']];
-  }
```
Comment (joeyorlando) [inline grafana-plugin/src/models/user/user.ts:88]: we already have a `UserStore.currentUser` `getter`.. this didn't seem to be referenced anywhere in the codebase (nor in `oncall-private`), I think it's safe to remove

**Comment by joeyorlando** [comment]
@teodosii there is still some minor styling polishing I believe, but I'll go ahead and merge this and we can follow up in a subsequent PR with that (+ address the two questions I had)

**Pull request merged**